### PR TITLE
Added config:status command.

### PR DESCRIPTION
**Pull request by danepowell**
This replaces #2726 

This is completely untested. I'm still trying to sort out a number of other Drush 9 bugs that prevent me from testing this.

### CODE REVIEW

**Comment by danepowell** [comment]
Okay this has now been tested and works great.

**Comment by weitzman** [comment]
Nice! 

Any chance you can exercise this command during [this test](https://github.com/drush-ops/drush/blob/master/tests/configTest.php#L26-L57)? Once thats in, rename that method to add Status.

If I'm pushing my luck too far just say so. No worries.

File: src/Drupal/Commands/config/ConfigCommands.php [hunk 1]
```diff
@@ -196,6 +197,86 @@ public function delete($config_name, $key = null)
     }
 
     /**
+     * Display status of configuration (differences between the filesystem configuration and database configuration).
+     *
+     * @command config:status
+     * @interact-config-label
+     * @param string $label A config directory label (i.e. a key in \$config_directories array in settings.php).
+     */
+    public function status($label = NULL)
```
Comment (weitzman) [inline src/Drupal/Commands/config/ConfigCommands.php]: $label needs an @param declaration in doxygen

File: src/Drupal/Commands/config/ConfigCommands.php [hunk 2]
```diff
@@ -196,6 +197,86 @@ public function delete($config_name, $key = null)
...
+    {
+        $directory = $this->getDirectory($label, '');
+        $storage = $this->getStorage($directory);
+        $change_list = $this->getChanges($storage);
+        if (empty($change_list)) {
+            $this->logger()->notice(dt('The active configuration is identical to the configuration in the export directory (!target).', array('!target' => $directory)));
+            return;
+        }
+        drush_print("Differences of the active config to the export directory:\n");
```
Comment (weitzman) [inline src/Drupal/Commands/config/ConfigCommands.php]: Use $this->output()->writeln().

File: src/Drupal/Commands/config/ConfigCommands.php [hunk 3]
```diff
@@ -196,6 +197,86 @@ public function delete($config_name, $key = null)
...
+     *   1. User-provided $directory.
+     *   2. Directory path corresponding to $label (mapped via $config_directories in settings.php).
+     *   3. Default sync directory
+     *
+     * @param string $label
+     *   A configuration directory label.
+     * @param string $directory
+     *   A configuration directory.
+     */
+    function getDirectory($label, $directory) {
```
Comment (weitzman) [inline src/Drupal/Commands/config/ConfigCommands.php]: Maybe $directory should default to null or ''?

File: src/Drupal/Commands/config/ConfigCommands.php (hunk 1 above)
Comment (danepowell) [reply src/Drupal/Commands/config/ConfigCommands.php]: There's already a `@param string $label ...` above this. Am I missing something?

Comment (weitzman) [reply src/Drupal/Commands/config/ConfigCommands.php]: oops, my mistake. @param usually comes right below @command.

**Comment by bircher** [comment]
it looks quite nice. a small nit pick maybe: I would use dependency injection instead of using `\Drupal::service('config.storage')`, `\Drupal::service('config.manager')` and `\Drupal::service('config.storage.sync')`.

**Pull request merged**